contracts/TokenTypeInterface.sol

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hubii Striim
+ *
+ * Compliant with the Hubii Striim specification v0.12.
+ *
+ * Copyright (C) 2017-2018 Hubii AS
+ */
+
+pragma solidity ^0.4.24;
+
+/**
+@title TokenTypeInterface
+@notice A base contract to handle different token types
+*/
+contract TokenTypeInterface {
```

Comment:
I think the notion of `Type` is too generic and also not relevant in the name of this interface. How about the simpler `TokenController`? Also note that none of the names of any of the other contracts we currently have contain `Interface`.

I would recommend that instead of _types_ of supported tokens we refer to _standards_. Thus the value set of supported standards will be ERC20, ERC721, ERC1155 etc. And somewhat related to this we could talk about _categories_ of supported tokens with value set of fungible and non-fungible.

Reply:
Standards! That was the word I didn't remember and didn't found in the chat history